Makefile

```diff
@@ -25,7 +27,7 @@ help:
 	@egrep "#\starget:" [Mm]akefile  | sed 's/\s-\s/\t\t\t/' | cut -d " " -f3- | sort -d
 
 # target: install - Install all files
-install: install-scripts install-conf install-systemd
+install: install-scripts install-conf install-systemd install-current-version-file
```

Comment:
This file is not relevant when installed as an AUR package. But the PKG can do an `rm` on it before building the package maybe